pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -197,6 +201,14 @@ func newFUSEFilesystem(ctx context.Context, devMinor uint32, opts *filesystemOpt
 
 // Release implements vfs.FilesystemImpl.Release.
 func (fs *filesystem) Release(ctx context.Context) {
+	fs.umounted = true
+	fs.conn.Abort(ctx)
+
+	fs.conn.fd.mu.Lock()
+	// Notify all the waiters on this fd.
+	fs.conn.fd.waitQueue.Notify(waiter.EventIn)
```

Comment:
Is this the right place to notify? Perhaps we should notify the waiters on the fs inside `Abort()`?